### DISCUSSION TITLE
Best way to add songs to a playlist in bulk?

### DISCUSSION THREAD

**Discussion by atiktak** [KRTirtho/spotube#902] [Q&A] [answered]
Hi! When I create a new playlist and want to add many songs from another playlist to it at once, is there a fast way to do this? I see that I can select multiple songs from my "Liked Tracks" and click "Add (x) to Playlist", but I don't see this option when selecting songs from another playlist. Is this currently not possible or am I missing something? Any help/tips are appreciated!

**Comment by KRTirtho** [comment, marked as answer]
Just long press any tracks in playlist/album which will select the tracks, the click on the vertical 3 dots top of the playlist/album beside sort button. There's an option "Add (<number of tracks>) to Playlist"

**Reply by atiktak** [reply]
Thanks for the reply. This is indeed the best way to do it.
I tried it again today and it seems the "Add () to Playlist" option just doesn't always show. I have opened an issue going more in detail on this.
